### DISCUSSION TITLE
Display Funktion

### DISCUSSION THREAD

**Discussion by msw-home** [pybricks/support#1895] [Q&A]
moin zusammen, gibt es eine Display funktion wie z.b. bei spike prime? ich möchte verschiedene Motorenwerte z.B grad anzeigen auf dem PC display.

**Comment by BertLindeman** [comment]
Hello Manfred,

Maybe I think too simple, but would that be a 
```python
print("motor angle:", motor.angle())
```

You will come back with a more difficult question I think 😄 

Bert

**Comment by msw-home** [comment]
Hallo Bert,keine schwierige Fragen, gibt nur schwierige Antworten! 😁Ja der Printbefehl wäre eine Möglichkeit, habe ich auch als erstes genutzt.Ich möchte meine diversen Motordaten aber auf einem Bildschirm anzeigen, die anzuzeigenden Daten vorzugsweise per Bluetooth übertragen.Die endgültige Version wäre dann, Motordaten am Bildschirm (Excel) ändern und zurück auf den Hub spielen.Ich komme halt aus dem Maschinenbau und Habe mittels SPS programmiert.Daher meine Spinnereien! 🙃😊Mit freundlichen Grüßen Manfred Schmidt-WintersteinWacholderweg 72A38239 Salzgitter E-Mail: ***@***.***:     0151 26091955Am 24.10.2024 um 16:18 schrieb Bert ***@***.***>:﻿
Hello Manfred,
Maybe I think too simple, but would that be a
print("motor angle:", motor.angle())
You will come back with a more difficult question I think 😄
Bert

**Reply by BertLindeman** [reply]
Ganz Gut Manfred,

Yet interesting! 
By far not a response to your question, but . . .
For debugging an xbox connection to a Technic hub I made a (wel, two) programs.
One transmits the xbox controls via bluetooth and the other prints the data in the pybricks terminal.

Nothing about hub to pc communication, but a bit about communicating values.
See [Transmit xBox controls from a TechnicHub and display them on another hub](https://github.com/orgs/pybricks/discussions/1883) 

Dinner time....

Bert

**Comment by msw-home** [comment]
Hallo Bert,

 

nur noch kurz zur Info.

Dieses Model habe ich mit KatzMike erstellt, er ist 17 Jahre und macht gerade sein Abi.

Er ist top fit im CAD Studio 2.0 mechanische Lego Konstruktion, wir haben zusammen seine Version 2.0 optimiert ist jetzt 3.0. Diese programmiere ich gerade mittels Pybricks, Blockprogramm, hierbei hast Du mich ja tatkräftig unterstützt.

 

https://rebrickable.com/mocs/MOC-194699/KatzMikeHD/6-a… <https://rebrickable.com/mocs/MOC-194699/KatzMikeHD/6-achsiger-roboterarm-30-mit-51515-6-axis-robotarm-with-51515/> LEGO MOC 6-achsiger Roboterarm 3.0 mit 51515, 6 axis Robotarm ...

08.09.2024 ... (2.0 => Lego Powered Up 42100) (3.0 => Lego Mindstorms 51515) Der ... Auf rebrickable werdet ihre die Legoteileliste des kompletten Roboterarms ...

 

 

Mit freundlichen Grüßen

 

 

Manfred Schmidt-Winterstein

Wacholderweg 72A

38239 Salzgitter

 

Tel:    0151 26091955

Mail:   ***@***.***> ***@***.***

 

Von: Bert ***@***.***> 
Gesendet: Thursday, 24 October 2024 16:18
An: pybricks/support ***@***.***>
Cc: msw-home ***@***.***>; Author ***@***.***>
Betreff: Re: [pybricks/support] Display Funktion (Discussion #1895)

 

Hello Manfred,

Maybe I think too simple, but would that be a

print("motor angle:", motor.angle())

You will come back with a more difficult question I think 😄

Bert
